packages/router/src/grpc/offscreen-client.ts

```diff
@@ -20,7 +20,6 @@ interface ContextsRequest {
 let active = 0;
 
 const activateOffscreen = async () => {
-  // @ts-expect-error: no types available yet
```

Comment:
Apparently, this doesn't trigger a TS error (at least, not anymore?), so TS complained:
<img width="460" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/97ce0e98-699e-401c-86b6-f1ca2b1085d3">